arangod/Graph/algorithm-aliases.h

```diff
@@ -86,7 +87,7 @@ using ShortestPathEnumerator = TwoSidedEnumerator<
                   VertexUniquenessLevel::GLOBAL, EdgeUniquenessLevel::PATH>>;
 
 template<class Provider>
-using WeightedShortestPathEnumerator = WeightedTwoSidedEnumerator<
+using WeightedShortestPathEnumeratorAlias = WeightedShortestPathEnumerator<
     WeightedQueue<typename Provider::Step>, PathStore<typename Provider::Step>,
     Provider,
     PathValidator<Provider, PathStore<typename Provider::Step>,
```

Comment:
`VertexUniquenessLevel::GLOBAL, EdgeUniquenessLevel::PATH` I think this can be improved. each ball uses its own `_foundVertices` set to keep track of where it was and edge uniqueness path is also just heating the universe, because we never create paths with duplicate edges.

Reply:
Very good point, will check right away.